secp256k1-sys/src/lib.rs

```diff
@@ -600,6 +643,38 @@ extern "C" {
                                  output_pubkey: *mut PublicKey,
                                  keypair: *const Keypair)
                                  -> c_int;
+    // Elligator Swift
+    #[cfg_attr(not(rust_secp_no_symbol_renaming), link_name = "rustsecp256k1_v0_9_0_ellswift_encode")]
+    pub fn secp256k1_ellswift_encode(
```

Comment:
All the code here and below uses different whitespace and formatting to the surrounding code, uniform would be better IMO.